### DISCUSSION TITLE
Add the option to Disable AI crew Conversations in Singleplayer mode

### DISCUSSION THREAD

**Discussion by Krzeszny** [FakeFishGames/Barotrauma#5156] [Ideas and suggestions]
The conversations are much too frequent, repeat too often, and very quickly become extremely annoying. 

Crew reports are useful, on the other hand, but get mixed up with conversations.

**Comment by mLuby** [comment]
Rather than yet another setting toggle, how about being able to tell a bot to be quiet _via the chat_?
> [Bot] Vasiliy Borodin: Can God see through leagues of ice?
> [Player] Crazy Ivan: quiet vasiliy
> [Bot] Vasiliy Borodin: I'll shut up now.
> …
> [Player] Crazy Ivan: speak borodin
> [Bot] Vasiiliy Borodin: I'm free to speak my mind.
> [Bot] Vasiiliy Borodin: Can God see through leagues of ice?

`/(quiet|speak).+(vasiliy borodin|vasiliy|borodin)/i` [matches](https://regex101.com/r/RySWmY/1) "quiet vasiliy" and "Please be quiet now, Captain Vasiliy Borodin." Different languages might have slightly different regexs.

**Reply by Krzeszny** [reply]
You can also add "stop talking", "shut up", "enough"... but telling each crew member, each mission, to be quiet will be even more annoying.

**Comment by pavel4win** [comment]
I am also annoyed by the chatter of bots in a single player game.
Maybe there is already a console command to prohibit bots from idle chatter?

**Comment by soltannis** [comment]
I get the fear in me of an attack every time I see them chat.

**Comment by KougamiYuriko** [comment]
I'm tired of AI robots chattering about the slightest injury. If I have 20 AIs, 4 doctors will repeat the same thing 4 times to each AI.
